server/bin/www.js

```diff
@@ -18,7 +18,7 @@ import config from '../configs/config'
  * Get port from environment and store in Express.
  */
 
-const port = normalizePort(config.app.port)
+const port = process.env.SERVER_PORT
```

Comment:
should we default the port to something like 3000 in case the variable isn't set?